docs/Documentation/UserGuide/6-System Tools/7-Monitor and Log Tools.md

```diff
@@ -357,3 +357,115 @@ Here are the file size statistics:
 |Timeseries Name| root.stats.file\_size.SCHEMA |
 |Reset After Restarting System| No |
 |Example| select SCHEMA from root.stats.file\_size.SCHEMA|
+
+## Performance Monitor
+
+### Introduction
+
+In order to grasp the performance of iotdb, we add this module to count the time-consuming of each operation. This module can statistic the avg time-consuming of each operation and the proportion of each operation fall into a time range. The output is in log_measure.log file. A output example is in below.  
```

Comment:
"time-consuming" -> "time consumption"
“statistic” -> "compute the statistics of"
"fall" -> "whose time consumption falls"
"A" -> "An"
"in below" -> "below"